### ISSUE TITLE
Sk89q/WE maven repo troubles

### ISSUE THREAD

**Issue by Muspah** [rlf/uSkyBlock#1131]
FYI: trying to build USB currently fails because of changes at the sk89q repository concerning worldedit-core. The maven-metadata in the repo makes maven believe that WE build 20180830 is the latest snapshot, where the actual latest build is from today.

**Comment by elohim-meth** [comment]
Trying to build latest master fails. Compiler complains: cannot find symbol:   class BlockVector3

**Comment by Muspah** [comment]
That's related to the worldedit maven problems indeed. There is a temporary workaround, edit your pom.xml file in the root folder (uSkyBlock parent pom) to look like:

https://hastebin.com/pulivonaji.xml

**Comment by elohim-meth** [comment]
Thanks for advice. I went a different way and manually corrected maven cache.

**Comment by asleeponduty** [comment]
If you're feeling lazy, here's a couple corrected maven.xmls. 
Edit the filenames and truncate everything past `.xml`
I noticed that the CoreMC repo used for the newer version of BStats also has out-of-date builds of Worldedit and Worldguard on it. I suspect that's part of the issue. 

In my case, I edited the files below after browsing the sk89q maven repo, and deleted the CoreMC xml files in the same directory. 

[maven-metadata-sk89q-repo.xml.worldedit-core.txt](https://github.com/rlf/uSkyBlock/files/2587349/maven-metadata-sk89q-repo.xml.worldedit-core.txt)
Belongs in `.m2\repository\com\sk89q\worldedit\worldedit-core\7.0.0-SNAPSHOT`
[maven-metadata-sk89q-repo.xml.worldguard-core.txt](https://github.com/rlf/uSkyBlock/files/2587350/maven-metadata-sk89q-repo.xml.worldguard-core.txt)
Belongs in `.m2\repository\com\sk89q\worldguard\worldguard-core\7.0.0-SNAPSHOT`
